sdk/eventhubs/azure-messaging-eventhubs/README.md

```diff
@@ -342,8 +342,8 @@ EventProcessorClient eventProcessorClient = new EventProcessorClientBuilder()
     .processEvent(eventContext -> {
         System.out.println("Partition id = " + eventContext.getPartitionContext().getPartitionId() + " and "
             + "sequence number of event = " + eventContext.getEventData().getSequenceNumber());
-        eventContext.updateCheckpoint(); // Update checkpoint for this partition.
     })
+    eventContext.updateCheckpoint(); // Update checkpoint for this partition.
```

Comment:
move before `})`

Reply:
Rolled it back, so hopefully it's fixed now. Sorry about that!